DevHome.sln

```diff
@@ -110,6 +110,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevSetupEngine.Test", "Hype
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DevSetupAgent.Test", "HyperVExtension\test\DevSetupAgent.Test\DevSetupAgent.Test.csproj", "{0E05A442-BDC7-43D4-A000-F8C986826716}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "HyperVExtension.HostGuestCommunication", "HyperVExtension\src\HyperVExtension.HostGuestCommunication\HyperVExtension.HostGuestCommunication.csproj", "{D759CD66-494C-4A00-8075-8B65A9891349}"

```

Comment:
This whole extension should go under the `tools` directory -- if anything, we want to move things around so there's less in the root directory, not more (there's even a GitHub issue for that). Alternatively, if we want to make the case that this isn't a "tool" since it doesn't have UI (or whatever) maybe we should have a generic "extensions" directory so that any future extensions can also go under it.

Reply:
will speak with Kristen about this offline

Reply:
Spoke to Kristen, We'll move our Hyper-v extension into a top level Extensions folder.